### DISCUSSION TITLE
Attach private files and new field to check report

### DISCUSSION THREAD

**Discussion by kokart** [globaleaks/globaleaks-whistleblowing-software#3403] [General]
Hello,

I've been testing the application and I have two doubts:

1. Is it possible enable a password to check the report? Similiar to check report of https://comp-eu.whistleblowernetwork.net/frontpage
2. Is it possible for a recepient attach "private documentation" to the report? We want to add some documentation which we don't want share with anyone. 

Thank you for your work,
Alfredo

**Comment by evilaliv3** [comment]
Thank you for your question @kokart 

1. At the moment we have not considered making it posbile to add a password on the report because a 16digit receipt is enough secure when combined with a good data retention policy. Would you please clarify the reasons behind the addition of this password?
2. this feature is considered and in line with our roadmap but still not available or scheduled for development.
